Source/WebGPU/WGSL/Metal/MetalFunctionWriter.cpp

```diff
@@ -193,18 +195,25 @@ void FunctionDefinitionWriter::visit(AST::Attribute& attribute)
 void FunctionDefinitionWriter::visit(AST::BuiltinAttribute& builtin)
 {
     // FIXME: we should replace this with something more efficient, like a trie
```

Comment:
Do we still plan to replace this with a trie? Or is the FIXME stale after this?

Reply:
I left this here because I'm not sure if we want a trie or not. That would be determined by profiling.